org.hdfgroup.HDFCompass.appdata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.hdfgroup.HDFCompass</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-HDF5=https://github.com/HDFGroup/hdf-compass/blob/develop/COPYING</project_license>
+  <developer id="org.hdf">
+    <name>The HDF Group</name>
+  </developer>
+  <name>HDFCompass</name>
+  <summary>Browse and plot HDF files</summary>
+  <url type="bugtracker">https://github.com/HDFGroup/hdf-compass/issues</url>
+  <url type="donation">https://www.hdfgroup.org/about-us/donate-to-the-hdf-group/</url>
+  <url type="help">https://hdf-compass.readthedocs.io/</url>
+  <url type="homepage">https://www.hdfgroup.org/2015/06/10/get-your-bearings-with-hdf-compass/</url>
+  <content_rating type="oars-1.1" />
+  <description>
+    <p>
+      HDF Compass is a viewer for HDF5 and related formats that can be extended through a plugin system.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://screenshots.debian.net/screenshot/hdf-compass/26632</image>
```

Comment:
did the link die? I'm getting 

![image](https://github.com/user-attachments/assets/8f313aae-34e1-44ba-8303-902663fd07b8)